sunbird-rc-esignet-integration-impl/pom.xml

```diff
@@ -0,0 +1,348 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>io.mosip.esignet.sunbirdrc</groupId>
```

Comment:
We should change the groupId to "io.mosip.esignet.plugin.sunbirdrc"

Reply:
the groupId of this plugin is inconsistent with groupid of other plugins. Is that ok?
mock-identity-plugin and mock-plugin have "io.mosip.esignet" as groupid.